assets/css/_late_view.scss

```diff
@@ -1,6 +1,147 @@
+$late-view-background-color: $color-bg-light;
+$late-view-horizontal-padding: 4rem;
+$late-view-panel-left-padding: 1.5rem;
+$late-view-width: 60rem;
+
 .m-late-view {
   @include modal-content;
+  background-color: $late-view-background-color;
+  padding: 2rem $late-view-horizontal-padding;
```

Comment:
Why define `$late-view-horizontal-padding` separately but not the `2rem` value?

Reply:
Oh is it just that `$late-view-horizontal-padding` needs to be referenced elsewhere?